ui/app/rif/components/chainAddresses.js

```diff
@@ -43,7 +44,7 @@ class ChainAddresses extends Component {
           resolvers,
         });
       });
-    const slipChainAddresses = Object.assign([], SLIP_ADDRESSES);
+    const slipChainAddresses = Object.assign([], _.orderBy(SLIP_ADDRESSES, ['name'], ['asc']));
```

Comment:
I think it's better if you just put an alias like, import _ as loadash from 'loadash'; on the import statement to avoid having _ all over the place when you use it like this.